app/views/need_help/_search_tips_body.html.erb

```diff
@@ -0,0 +1,36 @@
+<h4>When searching, it is helpful to keep a few things in mind:</h4>
+<ul>
+  <li>
+    Try many different terms. ‘ “NYU clubs”’ will produce different
```

Comment:
@jgpawletko I just noticed this in my own branch so thought I'd share. Let's replace these curvy quotes with straight ones, 

```
/[‘’]/'/ and /[“”]/"/
```